src/utils/localFormatted/localFormatted.ts

```diff
@@ -1,47 +1,35 @@
-export const locale =
-  (navigator.languages && navigator.languages[0]) ||
-  navigator.language ||
-  'en-US';
+export function formattedNumber(
```

Comment:
`formatWithAccuracy` would be better + you need to extend the implementation to make it possible to pad fraction part with zeroes, as discussed today

this is the case for the order book, where we need _1.12_ --> _1.12000_, given accuracy is 5